pylint/checkers/base_checker.py

```diff
@@ -234,7 +234,15 @@ def messages(self) -> list[MessageDefinition]:
             for msgid, msg_tuple in sorted(self.msgs.items())
         ]
 
-    def get_message_definition(self, msgid: str) -> MessageDefinition:
+    def get_message_definition(
+        self, msgid: str
+    ) -> MessageDefinition:  # pragma: no cover
```

Comment:
I'd prefer if we actually wrote a test for this or just keep it uncovered. There is no good reason this isn't covered other than we don't have a test for it.

We have similar test for deprecated stuff as well.